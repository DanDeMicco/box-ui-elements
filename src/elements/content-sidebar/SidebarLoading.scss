.be {
    .bcs-sidebar-loading {
        padding-top: 20px;
    }
}
