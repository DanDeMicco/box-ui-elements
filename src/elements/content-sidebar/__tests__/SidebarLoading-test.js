import * as React from 'react';
import { shallow } from 'enzyme';
import SidebarLoading from '../SidebarLoading';

describe('elements/content-sidebar/SidebarLoading', () => {
    const getWrapper = (props = {}) => shallow(<SidebarLoading {...props} />);

    test('should render the component', () => {
        const wrapper = getWrapper({
            tile: 'foo',
        });
        expect(wrapper).toMatchSnapshot();
    });
});
