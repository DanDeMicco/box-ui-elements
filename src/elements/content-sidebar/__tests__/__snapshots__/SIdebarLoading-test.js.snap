// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/content-sidebar/SidebarLoading should render the component 1`] = `
<SidebarContent>
  <LoadingIndicator
    className="bcs-sidebar-loading"
  />
</SidebarContent>
`;
